test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -318,15 +311,15 @@ def host_setup_for_tensorflow_inference(
         ec2_connection.run(
             (
                 f"pip install --no-dependencies --no-cache-dir "
-                f"'tensorflow-serving-api=={framework_version}' 'protobuf>=3.20,<3.21'"
+                f"'tensorflow-serving-api=={framework_version}'"
             ),
             hide=True,
         )
     else:
         ec2_connection.run(
             (
                 f"pip install --user -qq -U 'tensorflow<={framework_version}' "
-                f" 'tensorflow-serving-api<={framework_version}' 'protobuf>=3.20,<3.21'"
+                f" 'tensorflow-serving-api<={framework_version}'"
```

Comment:
Removed the protobuf version constraint because it prevents the matching version of tensorflow and tensorflow-serving-api from being installed. If we face protobuf-related version mismatch issues in the future, we can add a constraint at the necessary version back to this code.

Reply:
We run TF inference requests through a client-side python script on the host, in which we import a couple of functions from tf serving api.

https://github.com/aws/deep-learning-containers/blob/0a5db0d9e6ac8d0dd8928183efa8c33de3e02e85/test/dlc_tests/container_tests/bin/tensorflow_serving/example/mnist_client.py#L39-L40